app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class SessionsController < ApplicationController
+  def create
+    user = User.from_omniauth(request.env['omniauth.auth'])
+    session[:id] = user.id
+    flash[:success] = "Welcome #{user.name}"
+
+    redirect_to '/dashboard'
+  end
+
+  # def destroy
```

Comment:
The next step is to implement a logout feature and this action would be used for that.